src/Sylius/Bundle/ShopBundle/Resources/config/services.xml

```diff
@@ -14,7 +14,9 @@
 <container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
     <imports>
         <import resource="services/controller.xml" />
+        <import resource="services/context.xml" />
```

Comment:
By alphabetical order, `context` should be defined before `controller`

Reply:
Fixed.